### ISSUE TITLE
Fans are at full speed since V124

### ISSUE THREAD

**Issue by lmDarkLord** [Rem0o/FanControl.Releases#1053]
Ever since V124 my case fans have been going to 100% until FanControl opens.

If the curve that is set on the fans is turned off they immediately spin at 100%.

**Comment by Rem0o** [comment]
Have you rebooted since?  This can happen if somehow FanControl got force shut.

**Comment by lmDarkLord** [comment]
Hi there, I have reboot multiple times and even tried a BIOS update since I havent done that for a while, the fans were back to normal operation until FanControl rebooted, I'm now slowly transfering myself back to V122 to see if that fixes the issue.

**Comment by Rem0o** [comment]
The driver didn't change from 122 to 124.  So I'm not sure how that would help.

**Comment by lmDarkLord** [comment]
Is there a fix you recommend to this issue?

**Comment by Rem0o** [comment]
I would check what your fans are defaulting to when FanControl is off. Most likely your bios config, or your motherboard's software.

**Comment by lmDarkLord** [comment]
Yes I've checked that and found out that my fans will always max themselves out for some reason. I assumed it was because of FanControl because I updated it yesterday and it was the only update that occurred. I might be wrong though.

**Comment by Rem0o** [comment]
@r1y4h   FanControl had a problem on windows shutdown where it might/might not have the time to exit properly and release all ressources.  That's my only theory as to how it might happen for some people only. 

If you are stuck in that state, two things to try:

1: Make any change to a fan config in your bios to try and force it to apply a setting onto the lpc chip.
2: Clear CMOS

**Comment by r1y4h** [comment]
I already tried #1. Now even without the Fan Control and using Smart Fan control from Gigabyte and then after calibration, the fans would jump to almost 100% at 20% fan speed only. It wasn't like this before so something really has messed up after using Fan Control.

**Comment by lmDarkLord** [comment]
Hi there,
Did you try a full shutdown of the computer, back when I had this issue it turned out that i simply kept restarting the computer instead of shutting it down and that caused the issue.

**Comment by lmDarkLord** [comment]
Sorry about that.
Make sure to try a full shutdown of the computer, back when I had this issue I kept hitting the restart option instead of shutting it down and then opening it. 
hopefully this is what causes your issue

**Comment by r1y4h** [comment]
Hi @lmDarkLord thank you. The shutdown did the trick. I had the same experience as yours. Now my fans are back to normal. 
I came here looking for ways to fully control fan curve. It's just unfortunate that there is a side-effect, I really like Fan Control's interface.  Gigabyte software is not up to the task. I came from a ROG board which has a FanExpert is miles better than Gigabyte's Smart Fan control.

**Comment by lmDarkLord** [comment]
Glad that I could help, also this issue doesnt really happen unless you restart your computer - and even then the app should tune your fans back to normal once its open , they will still go to 100% when you shut your computer down but thats about it.
I barely experience this issue but even when it does happen its resolved very easily by a shutdown or as I mentioned the app itself.

**Comment by Rem0o** [comment]
Also, I made a change recently to force Windows to wait ( or let's say highly suggest ) for FanControl to completly close before proceeding with a shutdown. This should prevent stuff like what you are experiencing from happening.

**Comment by r1y4h** [comment]
I did see that, but of course I didn't know what it is for or if there would be any side effect. What I know is that I didn't save my changes so I guess that's fine. I just saw "Windows can't shutdown due to my Fan Control userconfig" something like that. Which is nothing unusual to be honest.

**Comment by Morphin3** [comment]
Me too.
1. Shut down ,start computer,speed is normal
2. Restart the computer,or just close the app,it will speed up to 100% when app is not opened

**Comment by Monsterardadrys** [comment]
I have the "opposite" problem. About every other shutdown (or sleepmode) sets the cpu and case fans to full speed until I do a reboot since I installed FanControl about a month ago. I don't think it started straight away, my sense is that it too a couple of days after I started to change fan curves. When I realized I just needed to reboot I have kind of accepted it. But I would love to know if there's a way to fix it.
